packages/database/src/browser/MigrationManager.js

```diff
@@ -0,0 +1,141 @@
+import { PGlite } from '@electric-sql/pglite';
+import path from 'path';
+import { getConnectionConfig } from './getConnectionConfig';
+
+const DEFAULT_MIGRATIONS_PATH = path.resolve(__dirname, '/packages/database/src/core/migrations');
+
+export class MigrationManager {
+  client = null;
+  config = null;
+
+  constructor(config = {}) {
+    this.config = {
+      ...config,
+      migrationsPath: config.migrationsPath || DEFAULT_MIGRATIONS_PATH,
+      logger: config.logger || {
+        info: message => console.log(message),
+        error: message => console.error(message),
+      },
+    };
+    this.client = new PGlite(getConnectionConfig);
+  }
+
+  async initialize() {
+    await this.createMigrationsTable();
+  }
+
+  async close() {
+    await this.client.close();
+  }
+
+  async createMigrationsTable() {
+    await this.client.exec(`
+      CREATE TABLE IF NOT EXISTS migrations (
+        id SERIAL PRIMARY KEY,
+        version INTEGER NOT NULL,
+        name VARCHAR(255) NOT NULL,
+        executed_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+      );
+    `);
+  }
+
+  async loadMigrationFiles() {
+    const migrationFiles = import.meta.glob('../../core/migrations/*.js', { eager: true });
+    const migrations = [];
+
+    const sortedMigrationFiles = Object.fromEntries(
+      Object.entries(migrationFiles).sort((a, b) => a[0].localeCompare(b[0])),
+    );
+
+    for (const [name, migrationModule] of Object.entries(sortedMigrationFiles)) {
```

Comment:
```suggestion
    for (const [name, migrationFile] of Object.entries(sortedMigrationFiles)) {
```
Although it's technically a module with named exports, I think it's slightly misleading as a "module" is generally thought of as something bigger (like the whole migration system is what I'd think of as the migration module)